bundles/org.openhab.core/src/main/java/org/openhab/core/library/types/DecimalType.java

```diff
@@ -49,6 +49,20 @@ public DecimalType(Number value) {
         this(bigDecimal(value));
     }
 
+    // TODO: Remove this constructors. They are still in place to maintain binary compatibility and will be removed once
```

Comment:
I still have a review comment. :wink: 

```suggestion
    // TODO: Remove these constructors. They are still in place to maintain binary compatibility and will be removed once
```